src/tools/tidy/src/deps.rs

```diff
@@ -62,6 +62,8 @@ const WHITELIST_CRATES: &[CrateVersion<'_>] = &[
 const WHITELIST: &[Crate<'_>] = &[
     Crate("adler32"),
     Crate("aho-corasick"),
+    Crate("annotate-snippets"),
+    Crate("ansi_term"),
```

Comment:
cc @rust-lang/compiler 	we're pulling in a new dependency here, and we have no control over it: `ansi_term` (`annotate-snippets` is also a new dep, but it is under the `rust-lang` org: https://github.com/rust-lang/annotate-snippets-rs).

Reply:
Two points in favour of allowing it: its license (MIT) is in our list of allowed licenses for external crates; and it is already a transitive dependency of our other core projects such as `cargo`.

As long as changes to the `ansi_term` are vetted as part of the `annotate-snippets` crate and the crate is locked to a specific patch version I’m in favour of the dependency.

Reply:
The usage of `ansi_term` in `annotate-snippets` seems pretty lightweight so far and we could probably switch to another one as well, if that's a concern.